src/traccuracy/loaders/_point.py

```diff
@@ -29,20 +31,25 @@ def load_point_data(
             Defaults to None.
         parent_column (str | None, optional): A reference to the parent node in the previous
             time frame. Defaults to "parent".
-        id_column (str | None, optional): Optional column used to specify node ids.
-            Defaults to None and the row index is used as the id
+        id_column (str, optional): Optional column used to specify node ids.
+            Defaults to "node_id"
         pos_columns (tuple[str], optional): A tuple of columns to use for position.
             Defaults to ("z", "y", "x").
         time_column (str, optional): The column to use for time. Defaults to "t".
+        seg_id_column (str | None, optional): Name of an optional column containing a segmentation
+            label id. Defaults to None.
         name (str | None, optional): Optional string to name/describe the dataset. Defaults to None.
         sep (str | None, optional): Passed to pd.read_csv to set the sep kwarg. Defaults to None.
+        no_parent_val (str | None, optional): The value used to indicate that a node
```

Comment:
I don't think you need to make this an argument. I think you can just document what the CSV should look like (I prefer -1 OR None, but up to you).